tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -46,6 +48,12 @@ func TestServiceEntry(t *testing.T) {
 			url:               "https://bing.com",
 			shouldBeReachable: false,
 		},
+		{
+			name:              "UNREACHABLE_bing.com_over_google_80",
```

Comment:
this test is redundant since we check for unreachable bing.com over *.bing.com:80 below

Reply:
I renamed this to UNREACHABLE_www.bing.com_over_google_80 as this tests www.bing.com, the other tests bing.com.